components/faq/questions.js

```diff
@@ -39,10 +39,10 @@ const questions = {
       answer: (
         <>
           Operation Code, much like software, is built from anywhere with an internet connection,
-          and is not based in one location. While we're headquartered in Portland, the entire
+          and is not based in one location. While we&apos;re headquartered in Portland, the entire
           organization is decentralized, including the board of directors and the core team. This
           allows us to more effectively serve the entire military community, whether they're
```

Comment:
missed one right here in `they're`

Reply:
Ok thank you!

Reply:
I fixed that and went through each single quote one at a time. I noticed on line 225 there is a concatenated string. I tried making it a single string but the eslint wouldn't allow the commit. Any suggestions?